web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -153,7 +146,14 @@ const formatPhoneNumber = (value: string): string => {
 export default function MyPage() {
   const [theme, toggleTheme] = useTheme();
   const navigate = useNavigate();
-  const [state, dispatch] = useReducer(profileReducer, initalState);
+  const [passwordDisabled, setPasswordDisabled] = useState(false);
+  const [phoneNumberDisabled, setPhoneNumberDisabled] = useState(false);
+  const [addressDisabled, setAddressDisabled] = useState(false);
+  const [titleAlert, setTitleAlert] = useState('');
+  const [textAlert, setTextAlert] = useState('');
```

Comment:
Please rename this referring to [this code](https://github.com/kookmin-sw/capstone-2023-40/pull/134/files#diff-6d0ce7f4b1dca2052a6177f49c5f2ab43f17a49651a9d7583fcac5d4ef35a899R101).